reframe/core/meta.py

```diff
@@ -162,8 +170,17 @@ def __prepare__(metacls, name, bases, **kwargs):
         namespace['variable'] = variables.TestVar
         namespace['required'] = variables.Undefined
 
-        # Directive to bind a free function into the class
         def bind(fn, name=None):
+            '''Directive to bind a free function to a class.
+
+            By default, the function is bound with the same name as the free
+            function. However, the function can be bound using a different name
+            with the ``name`` argument.
+
+            :param fn: external function to be bound to a class.
+            :param name: bind the function under a diferent name.
```

Comment:
We need a `.. versionadded:: 3.6.2` here.

Reply:
And actually this is not documented publicly. We need to add this file to the corresponding rst file.

Reply:
Good shout. The docs from the `run_before`, `run_after` and `require_deps` decorators was also missing. I have now rearranged this a bit and moved that to the `builtins` section of the docs.